lib/adapters/tool/testplane/index.ts

```diff
@@ -1,5 +1,6 @@
 import _ from 'lodash';
-import Testplane, {type Config} from 'testplane';
+import type Testplane from 'testplane';
```

Comment:
Testplane is not imported here, because this file is used also for hermione tool